app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :likes
+  has_many :comments
+
+  def return_most_post
+    Post.where(author_id: id).order(created_at: :desc).limit(3)
+  end
+end
+
+# first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+# second_user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher
+# from Poland.')
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: